web/modules/custom/usagov_wizard/usagov_wizard.module

```diff
@@ -228,6 +228,26 @@ function usagov_wizard_preprocess_html(&$variables) {
           $variables['wizardTitle'] = $wizard_title;
           $variables['wizardTitleUrl'] = $wizard_title_url;
         }
+        else {
+          // Lookup the top-most taxonomy parent
+          $topParent = $taxonomy_term;
+          while (!empty($topParent->parent)) {
+            $thisParent = Term::load($topParent->parent->target_id);
+            if (empty($thisParent)) {
+              break;
+            }
+            $topParent = $thisParent;
+          }
```

Comment:
we should reuse `MenuChecker::getTermParents()` here, which uses the taxonomy_term entity type managers method to `loadAllParents()` for a term.

Reply:
Ah good Idea - this is now resolved on this branch.